src/doc/unstable-book/src/compiler-flags/indirect-branch-cs-prefix.md

```diff
@@ -0,0 +1,19 @@
+# `indirect-branch-cs-prefix`
+
+The tracking issue for this feature is: https://github.com/rust-lang/rust/issues/116852.
+
+------------------------
+
+Option `-Zindirect-branch-cs-prefix` controls whether a `cs` prefix is added to
+`call` and `jmp` to indirect thunks.
+
+It is equivalent to [Clang]'s and [GCC]'s `-mindirect-branch-cs-prefix`. The
+Linux kernel uses it for RETPOLINE builds. For details, see
+[LLVM commit 6f867f910283]("[X86] Support ``-mindirect-branch-cs-prefix`` for
+call and jmp to indirect thunk") which introduces the feature.
```

Comment:
Is this rendered correctly? It looks like the syntax for making `"[X86] Support ``-mindirect-branch-cs-prefix`` for call and jmp to indirect thunk"` into the url of the link. You might want a space after the ].